generators/server/templates/src/main/java/package/service/mapper/_UserMapper.java

```diff
@@ -54,7 +53,7 @@ default User userFromId(<% if (databaseType == 'mongodb' || databaseType == 'cas
     }
 
     default Set<Authority> authoritiesFromStrings(Set<String> strings) {
-        return strings.stream().map(string -> {
+        return strings == null ? null : strings.stream().map(string -> {
```

Comment:
I guess it's the first time we make a such relation to a User:
EntityA <-*------*-> User

If we save EntityA, there is an NPE.. (So far we don't have this use case I suppose).

If we don't want to have a null value, it means that we populate the UserDTO with all the data but only the 'id' will be used (to persist the relation between EntityA and the User).